android/KMEA/app/src/main/res/values/strings.xml

```diff
@@ -211,12 +211,21 @@
     <!-- Context: Language Settings Keyboard uninstall strings -->
     <string name="keyboard_deleted_toast" comment="Notification when a keyboard is deleted">Keyboard deleted</string>
 
-    <!-- Context: Language Settings menu strings -->
+    <!-- Context: Language Settings menu strings. Removed in Keyman 18 -->
     <string name="enable_corrections" comment="Enable corrections from the suggestion banner">Enable corrections</string>
 
-    <!-- Context: Language Settings menu strings -->
+    <!-- Context: Language Settings menu strings. Removed in Keyman 18 -->
     <string name="enable_predictions" comment="Enable predictions from the suggestion banner">Enable predictions</string>
```

Comment:
If these are removed, can't we just delete them from the source?

Reply:
I've been keeping old strings in Crowdin for 1 more release cycle. This way the translated strings are still available for the 17.0 builds